spring-ai-chat/accelerator.axl

```diff
@@ -0,0 +1,53 @@
+engine {
+
+  TextPreprocessor()
+
+  if (#vectorStoreType != 'simple') {
+    Exclude({"**/SimpleVectorStoreConfig.java"}) 
+  }
+
+  applyTo("pom.xml") {
+    ReplaceText({
+      {text: "<groupId>com.vmware.tap.accelerators</groupId>", with: '<groupId>' + #groupId + '</groupId>'},
+      {text: "<artifactId>ai-chat</artifactId>", with: '<artifactId>' + #artifactId + '</artifactId>'}
+    })
+    if (#aiService == 'azureOpenAI') {
+      ReplaceText({{text: "spring-ai-openai-spring-boot-starter", with: 'spring-ai-azure-openai-spring-boot-starter'}})
+    }
+  }
+
+  applyTo("src/main/resources/application.properties") {
+    if (#aiService == 'openAI') {
+      ReplaceText({{text: "spring.ai.openai.model=gpt-3.5-turbo", with: 'spring.ai.openai.model=' + #openaiModel}})
+    }
+    if (#aiService == 'azureOpenAI') {
+      ReplaceText({
```

Comment:
Realised we now have access to `ReplaceSnippet` and `ReplaceRegex`.

The former is like ReplaceText for static text, but for just one snippet instead of several, which often simplifies the syntax.
The latter is for the regex case.